src/App.js

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={article.title} date={article.date} text={article.text} />
```

Comment:
```suggestion
  <Article { ...article } />
```
You could pass the whole `article` object to the `props` object

Reply:
ok thanks